geoapps/octree_creation/constants.py

```diff
@@ -99,6 +99,15 @@
             "main": True,
             "value": 500.0,
         },
+        "minimum_level": {
+            "enabled": True,
+            "group": "Basic",
+            "label": "Minimum refinement level.",
+            "main": True,
+            "min": 1,
+            "tooltip": "Largest octree cell level allowed (2^i x base_cell).",
```

Comment:
I don't understand the tooltip.  It's a 'minimum' level but the tooltip suggests it's imposing a maximum.  What is 'i', the value of minimum_level?

Reply:
Yea fair enough. It's the maximum cell size, which is the minimum refinement level.
How about "tooltip": "Minimum octree cell level allowed away from refinements (2^i x base_cell).",